pallets/funding/src/tests/4_community.rs

```diff
@@ -226,26 +233,82 @@ mod community_contribute_extrinsic {
 				));
 			});
 
-			// If we mint what we just used, and mint what we cannot use due to being a slash deposit, \
-			// then we can use make a contribution that uses the same amount that we originally bonded on the evaluation
-			let spent_usable_plmc = (BOB, bobs_usable_bond / 2).into();
-			let necessary_plmc_mint = (BOB, bobs_slashable_bond).into();
-			inst.mint_plmc_to(vec![spent_usable_plmc, necessary_plmc_mint]);
+			// Can use the full evaluation bond
 			let contribution_usdt = MockInstantiator::calculate_contributed_funding_asset_spent(
-				vec![(BOB, usable_ct + slashable_ct).into()],
+				vec![(CARL, usable_ct).into()],
 				ct_price,
 			);
 			inst.mint_foreign_asset_to(contribution_usdt.clone());
+			inst.execute(|| {
+				assert_ok!(Pallet::<TestRuntime>::community_contribute(
+					RuntimeOrigin::signed(CARL),
+					get_mock_jwt(CARL, InvestorType::Retail, generate_did_from_account(CARL)),
+					project_id,
+					usable_ct,
+					1u8.try_into().unwrap(),
+					AcceptedFundingAsset::USDT,
+				));
+			});
+
+			// An evaluator that did a bid but it was not accepted at the end of the auction, can use that PLMC for contributing
```

Comment:
Would be good to move the logic after this comment to a separate test. Just for clarity sake.